src/storage/index.ts

```diff
@@ -777,7 +777,7 @@ async function listFilesLoop(
  * List the set of files in this application's Gaia storage bucket.
  * @param {function} callback - a callback to invoke on each named file that
  * returns `true` to continue the listing operation or `false` to end it
- * @return {Promise} that resolves to the number of files listed, or rejects with an error.
+ * @return {Promise} that resolves to the number of files listed (excluding the last if ended early), or rejects with an error.
```

Comment:
> that resolves to the number of files listed (excluding the last if ended early), or rejects with an error.

Resolves to the total number of listed files.  If the call is ended early, the last file is excluded or the entire call rejects with an error.